Makefile

```diff
@@ -60,6 +60,10 @@ endif
 # use customized config file
 include $(config)
 
+ifndef $(USE_MKLDNN)
```

Comment:
I suggest making MKL_DNN default for linux platform first because MacOS need brew install llvm to build with MKL-DNN.

Reply:
Not making MKL-DNN default on MacOS as of now. Thanks for the suggestions!